test/system/src/main/java/io/pravega/test/system/framework/TestExecutorFactory.java

```diff
@@ -22,21 +25,28 @@
     public enum TestExecutorType {
         LOCAL,
         DOCKER,
+        K8s,
```

Comment:
For consistency and clarity, please use capital letters and the full name: `K8s` --> `KUBERNETES`.